package/gluon-mesh-vpn-wireguard/Makefile

```diff
@@ -0,0 +1,13 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-mesh-vpn-wireguard
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-mesh-vpn-wireguard
+  TITLE:=Support for connecting meshes via wireguard
+  DEPENDS:=+gluon-core +libgluonutil +gluon-mesh-vpn-core +wireguard +@GLUON_SPECIALIZE_KERNEL:KERNEL_TUN +wgpeerselector +libubus
```

Comment:
Seems like wireguard doesn't need tun. 

Can I just `GLUON_SPECIALIZE_KERNEL:KERNEL_TUN` by `GLUON_SPECIALIZE_KERNEL:KERNEL_WIREGUARD` ?

Reply:
If I'm not mistaken, this is not possible, as wireguard is not in-tree for this OpenWrt release, therefor it can't be built-in.